packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts

```diff
@@ -75,9 +75,7 @@ export class DbtGitProjectAdapter extends DbtLocalCredentialsProjectAdapter {
         dbtVersion,
         useDbtLs,
     }: DbtGitProjectAdapterArgs) {
-        const localRepositoryDir = tempy.directory({
-            prefix: 'git_',
-        });
+        const localRepositoryDir = fs.mkdtempSync('/tmp/git_');
```

Comment:
by default `mkdtempSync` creates a dir in a local folder (backend) , we need to use absolute paths to use /tmp (like was tempy was using by default)